spec/models/osbb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Osbb, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
please add a Osbb factory, and tests if that factory is valid

Reply:
Resolve